### PR TITLE
Fix rolechosenmessage

### PR DESCRIPTION
**Pull request by FlXlT**
Fixed the RoleChoseMessage bug.

You cannot call a view from an activity statically. 
Resolved by calling a static method from the Activity itself and adjust views there.

Should be manually tested.

### CODE REVIEW

**Comment by dalderliesten** [comment]
Code looks good, if the manual testing at the TU tomorrow goes well it has my approval.

**Comment by wouterz** [comment]
Manual testing will prove if it works 100%, software quality looks good. So will also approve after testing

**Comment by dmalarev** [comment]
code looks good

**Comment by jessetilro** [comment]
I believe I will still have to mess around with this while I implement the server-side validation of role allocation, but for now this looks like a good fix to me.

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/RoleActivity.java [hunk 1]
```diff
@@ -101,39 +101,28 @@ public void onSaveInstanceState(Bundle savedInstanceState) {
...
+                break;
+            case JANITOR:
+                janitorView.setEnabled(false);
+                break;
+            case SCIENTIST:
+                scientistView.setEnabled(false);
+                break;
+            default:
+                break;
+        }
```
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/RoleActivity.java:125]: A shame this was implemented using a switch statement. Will refactor while implementing and integrating server-side validation of role allocation.

**Pull request merged**